api/restHandler/BulkUpdateRestHandler.go

```diff
@@ -0,0 +1,193 @@
+	appWorkflowService      appWorkflow.AppWorkflowService
+	enforcerUtil            rbac.EnforcerUtil
+	envService              request.EnvironmentService
+	gitRegistryConfig       pipeline.GitRegistryConfig
+	dockerRegistryConfig    pipeline.DockerRegistryConfig
+	cdHandelr               pipeline.CdHandler
+	appCloneService         appClone.AppCloneService
+	materialRepository      pipelineConfig.MaterialRepository
+	policyService           security2.PolicyService
+	scanResultRepository    security.ImageScanResultRepository
+}
+
+func NewBulkUpdateRestHandlerImpl(pipelineBuilder pipeline.PipelineBuilder, Logger *zap.SugaredLogger,
+	chartService pipeline.ChartService,
+	propertiesConfigService pipeline.PropertiesConfigService,
+	dbMigrationService pipeline.DbMigrationService,
+	application application.ServiceClient,
+	userAuthService user.UserService,
+	teamService team.TeamService,
+	enforcer rbac.Enforcer,
+	ciHandler pipeline.CiHandler,
+	validator *validator.Validate,
+	gitSensorClient gitSensor.GitSensorClient,
+	ciPipelineRepository pipelineConfig.CiPipelineRepository, pipelineRepository pipelineConfig.PipelineRepository,
+	enforcerUtil rbac.EnforcerUtil, envService request.EnvironmentService,
+	gitRegistryConfig pipeline.GitRegistryConfig, dockerRegistryConfig pipeline.DockerRegistryConfig,
+	cdHandelr pipeline.CdHandler,
+	appCloneService appClone.AppCloneService,
+	appWorkflowService appWorkflow.AppWorkflowService,
+	materialRepository pipelineConfig.MaterialRepository, policyService security2.PolicyService,
+	scanResultRepository security.ImageScanResultRepository) *BulkUpdateRestHandlerImpl {
+	return &BulkUpdateRestHandlerImpl{
+		pipelineBuilder:         pipelineBuilder,
+		Logger:                  Logger,
+		chartService:            chartService,
+		propertiesConfigService: propertiesConfigService,
+		dbMigrationService:      dbMigrationService,
+		application:             application,
+		userAuthService:         userAuthService,
+		validator:               validator,
+		teamService:             teamService,
+		enforcer:                enforcer,
+		ciHandler:               ciHandler,
+		gitSensorClient:         gitSensorClient,
+		ciPipelineRepository:    ciPipelineRepository,
+		pipelineRepository:      pipelineRepository,
+		enforcerUtil:            enforcerUtil,
+		envService:              envService,
+		gitRegistryConfig:       gitRegistryConfig,
+		dockerRegistryConfig:    dockerRegistryConfig,
+		cdHandelr:               cdHandelr,
+		appCloneService:         appCloneService,
+		appWorkflowService:      appWorkflowService,
+		materialRepository:      materialRepository,
+		policyService:           policyService,
+		scanResultRepository:    scanResultRepository,
+	}
+}
+
+func (handler BulkUpdateRestHandlerImpl) GetExampleOperationBulkUpdate(w http.ResponseWriter, r *http.Request) {
+	response, err := handler.chartService.GetBulkUpdateRequestExample("deployment-template")
```

Comment:
this should come from front end and should match the apiVersion/kind, we can add subresource to it that is deploymentTemplate